src/client/hud.cpp

```diff
@@ -27,6 +27,7 @@
 #include "util/enriched_string.h"
 #include "irrlicht_changes/CGUITTFont.h"
 #include "gui/drawItemStack.h"
+#include "IMesh.h"
```

Comment:
```suggestion
#include <IMesh.h>
```
Same for any other file that is located outside of `src/`, such as `lib/` or `irr/`.
This is a non-functional change but it clarifies the origin.